### PR TITLE
new page for desktop accessibility

### PR DESCRIPTION
**Pull request by ElviraMustafina**
- example with NVDA speech viewer
- skipped the HiDPI issues section: HiDPI support is now included in JDK, and the workaround is no longer needed (and also no longer available in the newer version of NVDA)

### CODE REVIEW

File: topics/compose/compose-desktop-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                else -> "Clicked $count times"
+            }
+            Text(text, modifier = Modifier.align(Alignment.Center), fontSize = 24.sp)
+        }
+    }
+}
+```
+{initial-collapse-state="collapsed" collapsible="true" collapsed-title=".clickable { count += 1 } .semantics(mergeDescendants = true)"}
+
+<img src="compose-desktop-accessibility.png" alt="Custom button with semantic rules" width="600"/>
```
[the remark this answers is not in the document]
Comment (ElviraMustafina) [reply topics/compose/compose-desktop-accessibility.md]: I added a screenshot with Accessibility Inspector and a few details about NVDA and JAWS.

**Review by igordmn — APPROVED** [review]

File: topics/compose/compose-desktop-accessibility.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                else -> "Clicked $count times"
+            }
+            Text(text, modifier = Modifier.align(Alignment.Center), fontSize = 24.sp)
+        }
+    }
+}
+```
+{initial-collapse-state="collapsed" collapsible="true" collapsed-title=".clickable { count += 1 } .semantics(mergeDescendants = true)"}
+
+To test the application's accessibility on macOS, you can use [Accessibility Inspector](https://developer.apple.com/documentation/accessibility/accessibility-inspector)
```
Comment (zamulla) [inline topics/compose/compose-desktop-accessibility.md:67]: I feel that "test accessibility" is too broad here, maybe "test the custom semantics"?

**Review by zamulla — CHANGES_REQUESTED** [review]
Overall great, thanks! There are a couple of smaller issues I'd like to address.

File: topics/compose/compose-desktop-accessibility.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+| Platform | Accessibility status             |
+|----------|----------------------------------|
+| MacOS    | Fully supported                  |
+| Windows  | Supported via Java Access Bridge |
+| Linux    | Not supported                    | 
+
+## Enabling accessibility on Windows
+
+Accessibility on Windows is provided via Java Access Bridge, which is disabled by default.
+To enable it, run the following command:
```
[the remark this answers is not in the document]
Comment (ElviraMustafina) [reply topics/compose/compose-desktop-accessibility.md:15]: I'm actually not sure about this one.
@igordmn  Do we need JAB only for developers or users of our future app as well?

[the remark this answers is not in the document]
Comment (igordmn) [reply topics/compose/compose-desktop-accessibility.md:15]: This is about developer machine.

For user machine we have a task https://youtrack.jetbrains.com/issue/CMP-373/Accessibility-doesnt-work-on-Windows-when-we-create-a-native-distribution

[the remark this answers is not in the document]
Comment (igordmn) [reply topics/compose/compose-desktop-accessibility.md:15]: I found a way of enabling it on a user machine, but it needs verification:
https://youtrack.jetbrains.com/issue/CMP-373/Accessibility-doesnt-work-on-Windows-when-we-create-a-native-distribution#focus=Change-27-11130024.0-0

[the remark this answers is not in the document]
Comment (ElviraMustafina) [reply topics/compose/compose-desktop-accessibility.md]: Updated version: "To develop accessibility features on Windows, enable Java Access Bridge..."
I'll add info about the user's side once [CMP-373](https://youtrack.jetbrains.com/issue/CMP-373) is fixed.

**Review by zamulla — APPROVED** [review]
🎉

**Pull request merged**